generators/app/templates/template/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "watch": "onchange '**/*.d.ts' -e 'out/**' -- npm run build",
     "build": "typings bundle<%- ambient %> -o out",
-    "test": "ts-node test/test.ts"
+    "test": "ts-node test/test.ts",
+    "postinstall": "typings install"
```

Comment:
Never use `postinstall`, you're making every single person do this. Use `prepublish`.